src/components/ProgressBar/styles/ProgressBar.css.js

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components'
+import { getColor } from '../../../styles/utilities/color'
+import forEach from '../../../styles/utilities/forEach'
+
+/*
```

Comment:
Should this commented out code be removed?

Reply:
all done in 0c7afaa5